### ISSUE TITLE
[Accessibility- Script C 7.2] Notes- Some buttons are poorly implemented

### ISSUE THREAD

**Issue by mbensalem** [Meeds-io/meeds#3046]
**Environment:** Tribe

**Current behavior:**

Buttons in Notes have no labels:

- The button that opens the notes tree
- The button of filter
- The button of "Add page"
- The button of "Edite page"
-  The button of "Ask AI"
- The button "Open menu"

**Expected behavior:**

To add aria-label or title attribute to all buttons cited above

**Debate topic**

Which attribute must be used as a title for buttons (title or aria-label)?

**Comment by mbensalem** [comment]
Still reproduced on 7.2.x
